### PR TITLE
update to 3.19.3

### LINKED ISSUE #15: update to version 3.17.0

### PR DESCRIPTION
**Pull request by ocefpaf**
Replaces #7 
Closes #15 and #6

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,20 +21,22 @@ build:
 requirements:
   build:
     - python      # [win]
+    - readline 6.2*
     - vc 9  # [win and py27]
     - vc 10  # [win and py34]
     - vc 14  # [win and (py35 or py36)]
   run:
+    - readline 6.2*
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @msarahan we are still pinning to `sqlite 3.13.*` but, b/c `sqlite` is used to to build python and we may change that pinning in the future, I would like your opinion on the use of `readline` here.

**Comment by ocefpaf** [comment]
`OS X`:

```
+ conda inspect linkages -p /Users/travis/miniconda3/conda-bld/sqlite_1501090959969/_t_env sqlite
sqlite
------
ncurses-5.9-10:
    libtinfo.5.dylib (lib/libtinfo.5.dylib)
sqlite-3.19.3-0:
    libsqlite3.0.dylib (lib/libsqlite3.0.dylib)
system:
    libSystem.B.dylib (/usr/lib/libSystem.B.dylib)
    libedit.3.dylib (/usr/lib/libedit.3.dylib)
```

`Linux`:

```
+ conda inspect linkages -p /feedstock_root/build_artefacts/sqlite_1501087606775/_t_env sqlite
sqlite
------

ncurses-5.9-10:
    libncurses.so.5 (lib/libncurses.so.5)
    libtinfo.so.5 (lib/libtinfo.so.5)

readline-6.2-0:
    libreadline.so.6 (lib/libreadline.so.6)

system:
    libc.so.6 (/lib64/libc.so.6)
    libdl.so.2 (/lib64/libdl.so.2)
    libpthread.so.0 (/lib64/libpthread.so.0)
    linux-vdso.so.1 ()
```

Not sure if we need to do something extra to get `readline` on `OS X` :confused:

**Comment by ocefpaf** [comment]
Going ahead with this. If the addition of `readline` proves to be an issue we can revert it.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,17 @@
-{% set version="3.16.2" %}
-{% set year="2017" %}
-{% set version_split=version.split(".") %}
-{% set major=version_split[0] %}
-{% set minor=version_split[1]|int * 10 %}
-{% set bugfix=version_split[2]|int * 100 %}
-{% set version_coded=(major ~ (("%03d" % minor)|string) ~ (("%03d" % bugfix)|string)) %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Why was the version coding for the `url` dropped? :confused: This helps avoid issues like the one in PR ( https://github.com/conda-forge/sqlite-feedstock/pull/21 ).

**Pull request merged**